mitodl/micromasters
# Modified dashboard rest API to return proctorate exam grades

#### What are the relevant tickets?
closes #2915 

#### What's this PR do?
adds protectorate exam grades to the dashboard rest API 

#### How should this be manually tested?
Create some protectorate exam grades in the admin, then verify they appear in the dashboard rest api response.

#### Where should the reviewer start?
`dashboard/api.py`
